Create Partner-Billing-Guide (#30054)
Expensify/App: docs/articles/expensify-classic/expensify-partner-program/Partner-Billing-Guide.md

# FAQ

**Do I automatically get the special billing rate as an ExpensifyApproved! Partner?**
- Yes, when you join the ExpensifyApproved! program, you will automatically get the special billing rate. To join the ExpensifyApproved! Program, you need to enroll in ExpensifyApproved! University. 

**How can I check my billing details?**
- To check your billing details, be on the lookout for your detailed billing statement sent out at the end of each month. 

**Can I pass the bill to my clients?**
- Yes, you can pass the bill on to your clients. If you retain ownership of client workspaces in Expensify, you can re-bill your clients. If you’d like the client to own the billing of the Expensify, they can take over billing. 

**What if I don't want to use the Expensify Card?**
- If you prefer not to use the Expensify Card, your clients will be billed at $18/seat monthly.

**Why use client IDs?**
- Client IDs provide a streamlined method to identify and manage workspaces, especially beneficial when a client has multiple workspaces.

**Do I need a client ID for each Workspace?**
- Yes, if you want to ensure seamless identification and billing processes.

> ** Where can I see the Billing Receipts?**

Reviewer: ```suggestion
**Where can I see the Billing Receipts?**
```

---
title: Partner Billing Guide
description: Understand how Expensify bills partners and their clients
---

# Overview

> The ExpensifyApproved! Partner Program offers exclusive billing rates and features tailored for accountants to ensure seamless client management. If you are an accountant or consultant who recommends spend management solutions to your clients, becoming an ExpensifyApproved! Accountant may be a great certification for you. This guide will walk you through the unique billing perks available to ExpensifyApproved! partners, emphasizing the importance of understanding and managing client billing effectively. To learn what perks partners receive, check out the ExpensifyApproved! program details here.

Reviewer: > To learn what perks partners receive, check out the ExpensifyApproved! program details here.

Does `here` need to link to some page?

#  Get exclusive partner pricing

All ExpensifyApproved! Partners are automatically eligible for a special rate of $9/seat monthly, without an annual commitment when they adopt the Expensify Card. This provides flexibility as active users can vary. Here are the specifics on pricing for our Approved! Partners’ clients:

> - **Bundled pricing:** US Clients using the Expensify Card get up to a 50% discount, bringing their monthly bill to $9/seat. Reach out to your Partner Manager to discuss exclusive international pricing discount.s 

Reviewer: ```suggestion
- **Bundled pricing:** US Clients using the Expensify Card get up to a 50% discount, bringing their monthly bill to $9/seat. Reach out to your Partner Manager to discuss exclusive international pricing discount.
```

- **Unbundled pricing (or pay-per-use pricing):** Clients not using the Expensify Card are billed at $18/seat monthly.
- **No annual commitment:** Partners pay only for what they use, with no annual subscriptions required.

> - **Expensify Card revenue share:** All partners receive a 0.5% revenue share on Expensify Card transactions made by clients. This revenue share can be passed back to the client for an additional discount to offset their Expensify bill

Reviewer: ```suggestion
- **Expensify Card revenue share:** All partners receive a 0.5% revenue share on Expensify Card transactions made by clients. This revenue share can be passed back to the client for an additional discount to offset their Expensify bill.
```

# Understanding the billing process

> Expensify bills the owner of the expense policy for the activity on that policy. If accountants retain ownership of client policies, they receive the bill and can then re-bill their clients based on individual agreements.

Reviewer: ```suggestion
Expensify bills the owner of the expense workspace for the activity on that workspace. If accountants retain ownership of client workspaces, they receive the bill and can then re-bill their clients based on individual agreements.
```

Each month, Expensify will send a consolidated bill detailing:
- **Pay-per-use seats:** This is the number of active clients and their users for the month.
- **Expensify Card discount**: This amount reflects how much spend is put on your card, which then determines the discount for that month.

> - **Total monthly price:** This amount is the overall price of Expensify when using the Expensify Card discount to offset the cost of the pay-per-use seats

Reviewer: ```suggestion
- **Total monthly price:** This amount is the overall price of Expensify when using the Expensify Card discount to offset the cost of the pay-per-use seats.
```

> - **Policy list:** This is an overview of all client policies with their respective active seats.

Reviewer: ```suggestion
- **Workspace list:** This is an overview of all client workspaces with their respective active seats.
```

## Consolidated Domain Billing

> If your firm wishes to consolidate all Expensify billing to a single account, the Consolidated Domain Billing feature is your go-to tool. It centralizes payment for all group policies owned by any domain member of your firm.

Reviewer: ```suggestion
If your firm wishes to consolidate all Expensify billing to a single account, the Consolidated Domain Billing feature is your go-to tool. It centralizes payment for all group workspaces owned by any domain member of your firm.
```

### Activating Consolidated Domain Billing:
  1. Claim and verify your firm’s domain.
  2. Navigate to **Settings > Domains > [Domain Name] > Domain Admins** and set a **"Primary Domain Admin"** by using the drop down toggle to select an email address. 
  3. Enable **Consolidated Domain Billing** in the same section.

The Consolidated Domain Billing tool ensures that billing takes place under a single Expensify account associated with your firm. This eliminates the need to add multiple payment cards across various accounts to cover payments for multiple clients.

## Maintaining a Console of all clients:

> If your firm wants to have a console view of all client policies and domains, you will want to create a single, centralized login to manage all client policies and domains, such as accounting@myfirm.com. 

Reviewer: ```suggestion
If your firm wants to have a console view of all client workspaces and domains, you will want to create a single, centralized login to manage all client workspaces and domains, such as accounting@myfirm.com. 
```

>   1. Create a dedicated email address that will act as the universal policy owner, for example, accounting@myfirm.com.

Reviewer: ```suggestion
  1. Create a dedicated email address that will act as the universal workspsce owner, for example, accounting@myfirm.com.
```

>   1. Create a dedicated email address that will act as the universal policy owner, for example, accounting@myfirm.com.

Reviewer: ```suggestion
  1. Create a dedicated email address that will act as the universal workspace owner, for example, accounting@myfirm.com.
```

  2. Register this email with Expensify or your chosen platform and ensure it is verified and secured.

>   3. Within each client policy settings, add your centralized email (e.g. accounting@myfirm.com) as a policy admin.

Reviewer: ```suggestion
  3. Within each client workspace settings, add your centralized email (e.g. accounting@myfirm.com) as a workspace admin.
```

  4. Do the same with each client domain.

> 

Reviewer: ```suggestion
```

Reviewer 2: it doesn't affect the way article is rendered. So it's ok to not make this change

## Applying Client IDs to a bill 

Using client IDs for Optimized Billing in Expensify: A unique identifier feature for ExpensifyApproved! accountants. Streamline client Workspace recognition and make your billing process more efficient.

# How to assign a client ID to a workspace
  1. Log in to your account: Ensure you’re using an Approved! accountant account.
  2. Navigate to the desired Workspace: Go to **Settings > Workspaces > [Workspace Name] > Overview**.
  3. Input the identifier: Here, you can input an alphanumeric unique identifier for each client workspace.
**Note:** If a client has multiple workspaces, ensure each workspace has a consistent client ID.

# How to access and download billing receipts

> Accessing Billing **Settings: Go to Settings > Your Account > Payments > Billing History.**
> Download the Receipt: Click on **"Download Receipt CSV".**

Reviewer: ```suggestion
- Accessing Billing: **Settings: Go to Settings > Your Account > Payments > Billing History.**
- Download the Receipt: Click on **"Download Receipt CSV".**
```

> 

Reviewer: ```suggestion
```

# Deep Dive

> Using client IDs for all Workspaces: It's beneficial to use client IDs for all Workspaces to ensure each one is easily recognizable.
> Benefits of itemized billing receipts: Employing client IDs offers itemized billing by client, with each itemization detailing unique active users.

Reviewer: ```suggestion
- Using client IDs for all Workspaces: It's beneficial to use client IDs for all Workspaces to ensure each one is easily recognizable.
- Benefits of itemized billing receipts: Employing client IDs offers itemized billing by client, with each itemization detailing unique active users.
```

# FAQ

**Do I automatically get the special billing rate as an ExpensifyApproved! Partner?**
- Yes, when you join the ExpensifyApproved! program, you will automatically get the special billing rate. To join the ExpensifyApproved! Program, you need to enroll in ExpensifyApproved! University. 

**How can I check my billing details?**
- To check your billing details, be on the lookout for your detailed billing statement sent out at the end of each month. 

**Can I pass the bill to my clients?**

> - Yes, you can pass the bill on to your clients. If you retain ownership of client policies in Expensify, you can re-bill your clients. If you’d like the client to own the billing of the Expensify, they can take over billing. 

Reviewer: ```suggestion
- Yes, you can pass the bill on to your clients. If you retain ownership of client workspaces in Expensify, you can re-bill your clients. If you’d like the client to own the billing of the Expensify, they can take over billing. 
```

**What if I don't want to use the Expensify Card?**
- If you prefer not to use the Expensify Card, your clients will be billed at $18/seat monthly.

**Why use client IDs?**

> - Client IDs provide a streamlined method to identify and manage policies, especially beneficial when a client has multiple policies.

Reviewer: ```suggestion
- Client IDs provide a streamlined method to identify and manage workspaces, especially beneficial when a client has multiple workspaces.
```
